Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py

```diff
@@ -130,11 +130,6 @@ def get_alert_content(content_item, item_info, incident, sixgill_alerts_client):
         content_item['content'] = f'https://portal.cybersixgill.com/#/cve/{cve_id}'
         additional_info = item_info.get("additional_info", {})
         incident['CustomFields']['cve'] = cve_id
-        cybersixgillcvss31 = additional_info.get("nvd", {}).get("v3", {}).get("current")
```

Comment:
I think this change may break BC. @kgal-pan ?

Reply:
Not sure this is change causes a BC. From what I see, the call stack is as follows:

```
get_alert_content > add_sub_alerts_fields > item_to_incidents > fetch_incidents
```

This method (`get_alert_content`) is used only during a fetch and these fields' mapping were moved to the incoming mapper which is used during the fetch and [was requested to be set up as the default incoming mapper](https://github.com/demisto/content/pull/30787#discussion_r1398459300). I think that it shouldn't affect users in this case unless they don't set up the incoming mapper.